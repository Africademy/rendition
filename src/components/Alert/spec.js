/* globals expect, test */
import React from 'react'
import renderer from 'react-test-renderer'
import { Alert, Provider } from '../../../dist'

test('Alert renders correctly', () => {
  const component = renderer.create(
    <Provider>
      <Alert>An alert</Alert>
    </Provider>
  )
  let tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
