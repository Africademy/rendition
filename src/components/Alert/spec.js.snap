// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Alert renders correctly 1`] = `
.c3 {
  box-sizing: border-box;
}

.c4 {
  box-sizing: border-box;
  -webkit-flex: 1;
  -ms-flex: 1;
  flex: 1;
}

.c1 {
  font-family: 'Nunito',Arial,sans-serif;
  font-size: 14px;
  line-height: 1.5;
  box-sizing: border-box;
  -webkit-text-size-adjust: 100%;
  -ms-text-size-adjust: 100%;
  -moz-osx-font-smoothing: grayscale;
  -webkit-font-smoothing: antialiased;
}

.c2 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  -ms-flex-pack: justify;
  justify-content: space-between;
  margin: 0;
  border: 1px solid;
  min-height: 38px;
  padding: 8px 16px;
  border-radius: 3px;
  font-size: 16px;
  color: #3c3e42;
}

.c0 {
  font-family: 'Nunito',Arial,sans-serif;
  font-size: 14px;
}

.c0 h1,
.c0 h2,
.c0 h3,
.c0 h4,
.c0 h5,
.c0 h6,
.c0 button {
  font-family: CircularStd,Arial,sans-serif;
}

<div
  className="c0 c1"
>
  <div
    className="c2 c3"
  >
    
    <div
      className="c4"
    >
      
      An alert
    </div>
  </div>
</div>
`;
